### PR TITLE
Give a better error message on DN lookup failure

### PR DESCRIPTION
**Pull request by csordasmarton**
Write out current user name on DN lookup failure

### CODE REVIEW

File: libcodechecker/libauth/cc_ldap.py [hunk 1]
```diff
@@ -312,11 +312,9 @@ def auth_user(ldap_config, username=None, credentials=None):
                               account_scope)
 
         if user_dn is None:
-            LOG.error('Please check your LDAP server '
-                      'authentication credentials.')
+            LOG.error("Invalid credentials for user: '{0}'!".format(username))
```
Comment (gyorb) [inline libcodechecker/libauth/cc_ldap.py]: Please use a different logging format for optimization. You read about it [here](https://docs.python.org/2.6/library/logging.html#optimization)

**Comment by whisperity** [comment]
This outputs the LDAP service user's name in case such is configured, and not the username which came through the API for the user who tries to log in. Is this what we want?

**Comment by csordasmarton** [comment]
@whisperity I would like to see the user who tries to log in.

**Comment by whisperity** [comment]
Line 325:

```python
if not connection:
             LOG.info("User: " + username + " cannot be authenticated.")
```

*should* show the error message if the credentials are invalid, but the user exists.

The patch you are trying to add now will print the username if the DN lookup fails, which usually happens if the user does not exist. In this case, we are not sure yet if the password is valid or not.

**Pull request merged**